ValveKeyValue/ValveKeyValue.Test/Test Data/Text/serialization_types_expected.vdf

```diff
@@ -5,6 +5,7 @@
 		"VFloat"	"1234.5677"
 		"VInt"	"270544960"
 		"VLong"	"72623859790382856"
+		"VPointer"	"305419896"
```

Comment:
As discussed elsewhere, this seem to be making an assumption that isn't backed up by tier1. Do you have any example data files to support this?

Reply:
I don't, this is just how it ended up working because we convert IntPtr to int (for binary serialization).

Reply:
I'd be tempted to omit this from the test and just leave it as undefined behaviour.